python/lsst/sims/utils/WcsUtils.py

```diff
@@ -43,20 +45,13 @@ def _nativeLonLatFromPointing(lon, lat, lonPointing, latPointing):
     alpha = latPointing - 0.5*np.pi
     beta = lonPointing
 
-    ca=np.cos(alpha)
-    sa=np.sin(alpha)
-    cb=np.cos(beta)
-    sb=np.sin(beta)
-
-    v2 = np.dot(np.array([
-                          [1.0, 0.0, 0.0],
-                          [0.0, ca, sa],
-                          [0.0, -1.0*sa, ca]
-                          ]),
-                   np.dot(np.array([[cb, sb, 0.0],
-                                    [-sb, cb, 0.0],
-                                    [0.0, 0.0, 1.0]
-                                    ]), np.array([x,y,z])))
+    ca = np.cos(alpha)
+    sa = np.sin(alpha)
+    cb = np.cos(beta)
+    sb = np.sin(beta)
+
+    v2 = np.dot(np.array([[1.0, 0.0, 0.0], [0.0, ca, sa], [0.0, -1.0*sa, ca]]),
+                np.dot(np.array([[cb, sb, 0.0], [-sb, cb, 0.0], [0.0, 0.0, 1.0]]), np.array([x, y, z])))
```

Comment:
Actually it looks like some in other files might be too long, maybe double check the line lengths in all the files just to be safe.